### DISCUSSION TITLE
ERC20 Transaction

### DISCUSSION THREAD

**Discussion by A-M-H-F** [wevm/wagmi#1626] [Q&A]
Fixed!

**Comment by sniper365** [comment]
Any updates about this topic?

Really looking forward to it

**Comment by Cezarry** [comment]
Consider using `usePrepareContractWrite` and `useContractWrite` like that:

`
  const { config, error } = usePrepareContractWrite({
    address: tokenAddress,
    abi: erc20Abi,
    functionName: 'transfer',
    args: [recipient, amount]
  });
  const { write } = useContractWrite(config);
`
